### PR TITLE
[DjFS-11] Sign-Up Form Validation

### PR DESCRIPTION
**Pull request by MayowaShoyinka**
## Task: `[DJFS-11] Sign-Up Form Validation`
- `Created a domManipulation folder`

### CODE REVIEW

File: submissions/mayowaShoyinka/domManipulation/signupValidation.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+
+
+
+
+
```
Comment (adaugochi) [inline submissions/mayowaShoyinka/domManipulation/signupValidation.js:5]: remove the spaces above

File: submissions/mayowaShoyinka/domManipulation/signupValidation.js [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+let signupForm = document.getElementById('signupValidation'),
+    firstName = document.getElementById('firstName'),
+    lastName = document.getElementById('lastName'),
+    phoneNumber = document.getElementById('phoneNumber'),
+    emailAddress = document.getElementById('emailAddress'),
+    password = document.getElementById('password'),
+    confirmPassword = document.getElementById('confirmPassword')
+
+;
```
Comment (adaugochi) [inline submissions/mayowaShoyinka/domManipulation/signupValidation.js:14]: this semicolon should be on line 12 after the closing `)`

File: submissions/mayowaShoyinka/domManipulation/signupValidation.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+
+  function validateSignup(field,regdex,errorOne,errorTwo, event) {
```
Comment (adaugochi) [inline submissions/mayowaShoyinka/domManipulation/signupValidation.js:16]: kindly format this block of code properly

File: submissions/mayowaShoyinka/domManipulation/signupValidation.js [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+     event.preventDefault();
+     return false;
+   } else{
+     field.nextElementSibling.innerHTML = "";
+     return true;
+  }
+}
+
+ function signupValidate(event) {
+    let nameRegex =  /^[A-Za-z]+$/,
```
Comment (adaugochi) [inline submissions/mayowaShoyinka/domManipulation/signupValidation.js:32]: declare all these variables outside the function for better performance

**Review by adaugochi — CHANGES_REQUESTED** [review]
Also, you forget to handle the check for password and confirm password if they match

**Comment by MayowaShoyinka** [comment]
Changes Made, Thanks...Kindly Review.

**Comment by adaugochi** [comment]
The `J` in [DJFS-11]` should be in capital letter.  Kindly create a new PR. I will have to close this one

**Pull request closed without merging**